gensim/similarities/levenshtein.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 Vit Novotny <witiko@mail.muni.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+This module provides a namespace for functions that use the Levenshtein distance.
+"""
+
+import logging
+from math import floor
+
+# If python-Levenshtein is available, import it.
+# If python-Levenshtein is unavailable, ImportError will be raised in levdist.
+try:
+    import Levenshtein
+    LEVENSHTEIN_EXT = True
+except ImportError:
+    LEVENSHTEIN_EXT = False
+
+from gensim.similarities.termsim import TermSimilarityIndex
+
+logger = logging.getLogger(__name__)
+
+
+def levdist(t1, t2, max_distance=float("inf")):
+    """Get the Levenshtein distance between two terms.
+
+    Return the Levenshtein distance between two terms. The distance is a
+    number between <1.0, inf>, higher is less similar.
+
+    Parameters
+    ----------
+    t1 : {bytes, str, unicode}
+        The first compared term.
+    t2 : {bytes, str, unicode}
+        The second compared term.
+    max_distance : {int, float}, optional
+        If you don't care about distances larger than a known threshold, a more
+        efficient code path can be taken. For terms that are clearly "too far
+        apart", we will not compute the distance exactly, but we will return
+        `max(len(t1), len(t2))` more quickly, meaning "more than
+        `max_distance`".
+        Default: always compute distance exactly, no threshold clipping.
+
+    Returns
+    -------
+    int
+        The Levenshtein distance between `t1` and `t2`.
+
+    """
+    if not LEVENSHTEIN_EXT:
```

Comment:
make it simpler, just import here without try/catch (`*_EXT` have sense only for testing)